server/services/session_service.py

```
@@ -208,6 +208,17 @@ def open_session(name: str) -> Dict[str, Any]:
         logger.warning("Could not auto-set broadcast to live on session open: %s", exc)
 
+    # Webhook v2 — fire-and-forget on_session_start.
+    try:
+        from . import webhook as webhook_svc
+
+        webhook_svc.webhook_service.emit(
+            "on_session_start",
+            {"id": sid, "name": name, "started_at": now},
+        )
+    except Exception:
+        pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)